tests/unit/httpd_test.cc

```diff
@@ -2178,3 +2178,45 @@ SEASTAR_TEST_CASE(test_client_close_connection) {
         }
     });
 }
+
+static constexpr http::status_type my_http_status(555);
+static const auto my_http_status_name = http::bind_status_name(my_http_status, "My Super Error");
```

Comment:
Maybe define

```c++
status_type::status_type(int v, std::string_view name)
    : status_type(v)
{
    bind_status_name(name);
}
```

for one-line registration?

Reply:
That'd make only this `(int v, std::string_view name)` constructor non-constexpr, and why is _that_ a problem? Those custom status codes will behave like the "my_http_status_name" from this patch does